subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_alerts

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig NRF_CLOUD_ALERTS
+	bool "nRF Cloud Alert System"
+
+if NRF_CLOUD_ALERTS
+
+config NRF_CLOUD_ALERTS_SEQ_IF_NO_TIME
+	bool "Fall back to sequence number if time not available"
+	default y if DATE_TIME
```

Comment:
is this really just to produce an error message if an alert is sent without a timestamp?
i think you'd always want to send the alert (since it's probably important) even if there is no timestamp available.